sailthru/sailthru_content/services/sailthru_translation_service.py

```diff
@@ -43,7 +43,7 @@ def _create_course_vars(self, course, course_run, url, site_name, program_dictio
         seat_priority = ['credit', 'professional', 'verified', 'no-id-professional', 'audit', 'honor']
         sailthru_content_vars = {
             'course_run': True,
-            'marketing_url': url,
+            'marketing_url': url.split('?', 1)[0],
```

Comment:
Use the `exclude_utm=1` parameter on the API. See https://prod-edx-discovery.edx.org/api-docs/#!/api/Course_list.